sourced/ml/algorithms/uast_struct_to_bag.py

```diff
@@ -5,6 +5,8 @@
 
 
 class Uast2StructBagBase(Uast2BagBase):
+    SEP = "|"
```

Comment:
Do you think ">" is a better separator? We kind of point to the next node.

Reply:
makes sense